lms/djangoapps/instructor/views/instructor_dashboard.py

```diff
@@ -27,6 +27,7 @@
 from xblock.field_data import DictFieldData
 from xblock.fields import ScopeIds
 
+from cms.djangoapps.contentstore.config.waffle import ENABLE_PAGES_AND_RESOURCES_MICROFRONTEND
```

Comment:
Would it be better to also move this flag to `openedx/core/djangoapps/discussions/config/waffle.py`, so we have a centralize location for the flag for both the LMS and Studio imports?